lib/scope.go

```diff
@@ -44,54 +47,65 @@ func (s *scope) ActiveProfile() *profile.Profile {
 	return s.pro
 }
 
+// Bus returns the event bus
+func (s *scope) Bus() event.Bus {
+	// TODO(dustmop): Filter only events for this scope.
+	return s.inst.bus
+}
+
+// Config returns the config
+func (s *scope) Config() *config.Config {
+	return s.inst.cfg
+}
+
```

Comment:
alphabetized these methods, was having trouble quickly figuring out if a method existed on scope at a glance, so this was a solution.